test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
```

Comment:
You probably also want to test the response if a user goes directly to the callback url without going through github.